operators/multiclusterobservability/main.go

```diff
@@ -127,6 +128,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := imageregistryv1alpha1.AddToScheme(scheme); err != nil {
```

Comment:
do we need to use filteredcache to handle imageregistry resource?

Reply:
@marcolan018 discuss with @morvencao, we do not need filteredcache for imageregistry resource.

Reply:
Looks like the `imageregistry` resource don't have fix label and field for us to filter.
Also given that the `imageregistry` resource should be at most one in one cluster, so we don't need to filter it for cache, IMO.